x-pack/plugins/apm/public/components/fleet_integration/apm_policy_form/settings_definition/tls_settings.ts

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { i18n } from '@kbn/i18n';
+import { PackagePolicyVars, SettingDefinition } from '../typings';
+import {
+  isSettingsFormValid,
+  OPTIONAL_LABEL,
+  REQUIRED_LABEL,
+} from '../settings_form/utils';
+
+const TLS_ENABLED_KEY = 'tls_enabled';
```

Comment:
Eventually we may want to define these keys in `common` since they are also used in the config mapping: https://github.com/elastic/kibana/blob/master/x-pack/plugins/apm/server/lib/fleet/get_apm_package_policy_definition.ts#L170